README.md

```diff
@@ -4,8 +4,8 @@
 
 | Type | Link       | 
 |------|------------|
-| prod | <https://proj-gauchoride.dokku-00.cs.ucsb.edu/> | 
-| qa | <https://proj-gauchoride-qa.dokku-00.cs.ucsb.edu/>  | 
+| prod | <https://proj-gauchoride.dokku-03.cs.ucsb.edu/> | 
+| qa | <https://proj-gauchoride-qa.dokku-03.cs.ucsb.edu/>  | 
```

Comment:
It's typically considered good practice to try to not combine different concerns in the same PR; i.e. consider putting this in a separate PR next time.    

This time, though, I'll assign an extra 5 points for it in this PR.

Reply:
On second thought, given that this PR is big, complicated, and probably needs some work to be ready to merge, I'm going to encourage you to split this out into it's own PR for 5 points.

Reply:
created a new pr for this